plugins/fda_drugs/uploader.py

```diff
@@ -71,5 +80,6 @@ def get_mapping(self) -> dict:
             "te_code": {"type": "keyword"},
             "rld": {"type": "keyword"},
             "reference_standard": {"type": "keyword"},
```

Comment:
I cannot tell for sure, are `rld` and `rs` just boolean values?

Reply:
Yeah on the FDA drugs website they are either 0 (falsy) or 1 (truthy). The parser was failing before because everything was string so it was erroneously thinking that "0" -> truthy. After the conversion the values should be entirely boolean

Reply:
I updated both of those fields to be of type boolean now to better represent them in the elasticsearch mapping